app/controllers/users_controller.rb

```diff
@@ -20,4 +22,12 @@ def destroy; end
   def admin?
     current_user.admin?
   end
+
+  def set_user
+    @user = User.find(params[:id])
+  end
+
+  def redirect_non_admin_users
```

Comment:
@michellemhey : @darrendc and I assumed you probably came up with something similar for your #index story, it would be great to cross reference to ensure consistency. Curious to hear your thoughts.